core/src/test/scala/unit/kafka/log/LogTest.scala

```diff
@@ -66,7 +66,7 @@ class LogTest extends JUnitSuite {
     val set = TestUtils.singleMessageSet("test".getBytes)
 
     val logProps = new Properties()
-    logProps.put(LogConfig.SegmentMsProp, (1 * 60 * 60L): java.lang.Long)
+    logProps.put(LogConfig.SegmentMsProp, 1 * 60 * 60L: java.lang.Long)
```

Comment:
Even though this still works, I prefer the way it was done before (we do the `java.lang.Long` annotation on the whole expression).